src/libstd/collections/mod.rs

```diff
@@ -309,7 +309,7 @@
 //! }
 //! ```
 
-#![experimental]
+#![stable]
 
```

Comment:
Ooops, you're right, the `pub use self::hash_map::HashMap` threw me off here. This PR does include stabilizations of the facades in `libcollections/lib.rs`, though.